app/components/ui/notification-stream/component.js

```diff
@@ -42,7 +42,12 @@ export default Component.extend({
         const self = this;
 
         const event = self.get('selectedEvent');
-        self.get('store').findRecord('job', event.json.data.resource.jobs[0]).then(job => {
+        if (event.json.data.resource.jobs != null) {
+            var jobId = event.json.data.resource.jobs[0];
+        } else {
+            var jobId = event.json.data.resource._id;
```

Comment:
```suggestion
            let jobId = event.json.data.resource._id;
```

We don't use `var` anymore, as is not properly block scoped. Prefer to use `let` instead, where possible.

See https://blog.usejournal.com/awesome-javascript-no-more-var-working-title-999428999994